tests/cypress/e2e/spec.cy.ts

```diff
@@ -32,4 +33,33 @@ describe('Source configuration', () => {
     deleteCmkHost(hostName);
     activateCmkChanges('cmk');
   });
+
+  it('create a new time-usage panel', () => {
+    Cypress.config('defaultCommandTimeout', 30000);
+
+    createCmkHost(hostName);
+    activateCmkChanges('cmk');
+
+    cy.visit('/');
+    cy.get('input[name="user"]').type(Cypress.env('grafanaUsername'));
+    cy.get('input[name="password"]').type(Cypress.env('grafanaPassword'));
+    cy.get('[aria-label="Login button"]').click();
+
+    cy.visit('/dashboard/new');
+    cy.get('button[aria-label="Add new panel"]').click();
+
+    cy.get('input[id="react-select-7-input"]').type('Host name{enter}'); // Filter -> Hostname
+    cy.get('input[id="input_Host"]').click();
+    cy.contains(hostName).click({ force: true }); // Hostname -> <current host>
+
+    cy.get('input[id="input_Template"]').type('{downArrow}{enter}'); // Template -> Time usage by phase
+
+    cy.get('button[title="Apply changes and save dashboard"]').contains('Save').click({ force: true });
+    cy.get('input[aria-label="Save dashboard title field"]').type(' ' + randID);
+    cy.get('button[aria-label="Save dashboard button"]').click();
+    cy.reload();
+
```

Comment:
do we need to verify that something actually happened? or do we just check for javascript errors?